MAINTAINERS.md

```diff
@@ -0,0 +1,36 @@
+<!-- This is a generated file. Please do not edit directly -->
+
+# Maintainers
+
+This file lists how the ChefDK project is maintained. When making changes to the system, this
+file tells you who needs to review your patch - you need a simple majority of maintainers
+for the relevant subsystems to provide a :+1: on your pull request. Additionally, you need
+to not receive a veto from a Lieutenant or the Project Lead. ChefDK also has an internal Clients
+Pool team of all members have maintainership authority.
```

Comment:
@tduffield No reason not to, just wanted to get feedback before doing so. :+1: 

@jtimberman With regards to the frequency of change, I can't say for sure. However with regards to people whom work for Chef getting moved to another team which is not focused on the work, that is the current state of affairs for the most part, and I do see it happening again in the future.